packages/samples/atomic-react/src/App.tsx

```diff
@@ -43,6 +50,7 @@ const App: FunctionComponent = () => {
         <FoldedResultListWithCustomChildrenPage />
       )}
       {page === INSTANT_RESULTS_PAGE && <InstantResultsPage />}
+      {page === TABLE_RESULTS_LIST_PAGE && <TableResultListPage />}
```

Comment:
It's "result list" and "instant results" but not "results list", that would become confusing fast 😄 

(true for other places in the code as well)